### ISSUE TITLE
Missing mandatory schemaLanguage parameter for DescribeRecord

### ISSUE THREAD

**Issue by bermud** [opengeospatial/ets-csw202#10]
When testing this service
http://ogc.intergraph.com/CSW2015-CITE/Service.svc/get?request=GetCapabilities&service=CSW

The following assertion error is found:
```
Assertion failed: Missing mandatory schemaLanguage parameter for DescribeRecord, "http://www.w3.org/XML/Schema"
```

However, the "XMLSCHEMA" used is exactly the same as in the csw specification (page 137)

 "The schemaLanguage parameter is used to specify the schema language that should be used to describe the specified types. The default value is XMLSCHEMA, which indicates that the XML-Schema, schema description language shall be used. Other schemas languages are possible as long as the required schemaLanguage values are declared in the Capabilities document."

Is this a bug?

**Comment by bermud** [comment]
@rjmartell can you please take a look at this issue. It seems the spec default value is XMLSCHEMA and not the URI representing the XML schema. I think we should allow for both. Can you please make the changes?

```
<ows:Parameter name="schemaLanguage">
<ows:Value>XMLSCHEMA</ows:Value>
</ows:Parameter>
```

but I see this from the RI is passing

```
<ows:Parameter name="schemaLanguage">
<ows:Value>http://www.w3.org/XML/Schema</ows:Value>
<ows:Value>http://www.w3.org/TR/xmlschema-1/</ows:Value>
<ows:Value>http://www.w3.org/2001/XMLSchema</ows:Value>
</ows:Parameter>
```

**Comment by rjmartell** [comment]
The default value declared in [CSW-discovery.xsd](http://schemas.opengis.net/csw/2.0.2/CSW-discovery.xsd) is "http://www.w3.org/XML/Schema". This is also shown in the listing in 07-006r1, cl. 10.6.3 so the entry in Table 62 appears to be incorrect (also the text in 10.6.4.4).

This constraint is checked in the [Schematron schema](https://github.com/opengeospatial/ets-csw202/blob/master/src/main/resources/sch/csw/2.0.2/Capabilities.sch) used to validate the capabilities document.

**Comment by bermud** [comment]
@rjmartell. Thank you. I see the schema declares 
<xsd:attribute name="schemaLanguage" type="xsd:anyURI" use="optional" default="http://www.w3.org/XML/Schema"/>

**Comment by lorebiga** [comment]
@bermud , @rjmartell , please note that this issue had been already reported (by Stan), discussed, and fixed (by myself) as [issue #901](https://portal.opengeospatial.org/?m=projects&a=view&project_id=85&tab=5&act=details&issue_id=901).
I had come to the very same conclusion.
